WireAPI/Sources/WireAPI/Public/Models/BackendInfo.swift

```diff
@@ -0,0 +1,41 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+
+/// Metadata about the local backend.
+
+public struct BackendInfo {
```

Comment:
I always add a space to add some vertical separation to avoid crowding the type declaration or signature (makes it easier to read for me) but I'm not precious about it so if there's an agreed convention then I'll happily stick to that.